### PR TITLE
Add Chinese translations

### PR DESCRIPTION
**Pull request by kluo**
@caesar2164 @stvstnfrd

### CODE REVIEW

File: conf/locale/zh_CN/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,12905 @@
+# #-#-#-#-#  tos.po (stanford-openedx)  #-#-#-#-#
+# Stanford OpenEdX translation file
+# Copyright (C) 2018 Stanford University
+# This file is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
+# Stanford OpenEdX Team, 2018
+# 
+# Translators:
```
Comment (stvstnfrd) [inline conf/locale/zh_CN/LC_MESSAGES/django.po:7]: Why are there so many `Translators` stanzas?

**Review by stvstnfrd — COMMENTED** [review]
shrug

My question aside, I can't really do much to review the content itself, so if it works, it works.

**Review by caesar2164 — APPROVED** [review]
Since i read mandarin perfectly `</sarcasm>`, I'm 👍

File: conf/locale/zh_CN/LC_MESSAGES/django.po (hunk 1 above)
Comment (kluo) [reply conf/locale/zh_CN/LC_MESSAGES/django.po:7]: When merged with msgcat, the file headers just get joined as is

**Pull request merged**